### PR TITLE
Delete notification node methods and documentation

### PR DESCRIPTION
**Pull request by ofoucherot**
### Description

This PR adds:
- the liveblocks-node methods to delete a user's inbox notifications and to delete all user's inbox notifications.
- New REST endpoints documentation.
- New liveblocks-node methods documentaiton.

#### Related issue(s)

https://linear.app/liveblocks/project/delete-notification-5e80d69f613a/overview

### CODE REVIEW

File: packages/liveblocks-node/src/__tests__/client.test.ts [hunk 1]
```diff
@@ -1164,4 +1164,48 @@ describe("client", () => {
...
+      )
+    );
+
+    const client = new Liveblocks({ secret: "sk_xxx" });
+
+    await expect(
+      client.deleteInboxNotifications({
+        userId,
+      })
+    ).resolves.toBeUndefined();
```
Comment (FlowFlorent) [inline packages/liveblocks-node/src/__tests__/client.test.ts:1209]: maybe add a test to assert that it throws when the server returns something else than 2XX?

**Review by FlowFlorent — APPROVED** [review]
Looks good!
Just one small comment

**Review by marcbouchenoire — APPROVED** [review]
🙌

Can you keep this PR open until the React part is also done? Or open a new `2.3` branch? (the release will probably also contain the new text components)

**Review by CTNicholas — APPROVED** [review]

**Comment by marcbouchenoire** [comment]
@ofoucherot I started working on the React side and there we have the existing `useMarkAllInboxNotificationsAsRead` hook so for the "delete all notifications" one I'm going to follow the convention and name it `useDeleteAllInboxNotifications`, could you rename the `@liveblocks/node` method from `deleteInboxNotifications` to `deleteAllInboxNotifications`?

**Pull request merged**